frame/executive/src/lib.rs

```diff
@@ -148,7 +148,8 @@ pub type OriginOf<E, C> = <CallOf<E, C> as Dispatchable>::Origin;
 /// - `Block`: The block type of the runtime
 /// - `Context`: The context that is used when checking an extrinsic.
 /// - `UnsignedValidator`: The unsigned transaction validator of the runtime.
-/// - `AllPallets`: Tuple that contains all modules. Will be used to call e.g. `on_initialize`.
+/// - `AllPallets`: Tuple that contains all pallets including frame system pallet. Will be used to
```

Comment:
```suggestion
/// - `AllPalletsWithSystem`: Tuple that contains all pallets including frame system pallet. Will be used to
```

Reply:
fixed in https://github.com/paritytech/substrate/pull/10043/commits/b41e9359a4a156fdf42c91db031ef25456081278